fs_storage/models/fs_storage.py

```diff
@@ -116,6 +116,11 @@ def __init__(self, env, ids=(), prefetch_ids=()):
         compute="_compute_json_options",
         inverse="_inverse_json_options",
     )
+
+    eval_options_from_env = fields.Boolean(
+        string="Evaluate options values starting with $ from environment variables"
```

Comment:
That is a very long label :)
```suggestion
        string="Resolve env vars",
        help="Resolve options values starting with $ from environment variables"
```